.pre-commit-config.yaml

```diff
@@ -21,6 +24,12 @@ repos:
     hooks:
     -   id: check-hooks-apply
     -   id: check-useless-excludes
+-   repo: https://github.com/pre-commit/mirrors-pylint
+    rev: v1.9.1
+    hooks:
+    -   id: pylint
+        additional_dependencies: [pylint-venv]
+        args: ['--disable=all']
```

Comment:
This disables it initially as we need to turn it on and work thru some issues in regards to lint errors.